### PR TITLE
Some of the critical issues fixed here

### PR DESCRIPTION
**Pull request by vradu10**
- menu colors
- icons

### CODE REVIEW

File: _includes/_favicon.html [hunk 1]
```diff
@@ -4,27 +4,27 @@
 *   
 {% endcomment %}
 {% comment %} Regular Old-Skool Favicon: {% endcomment %}
-	{% if site.favicon-32x32 %}<link rel="icon" sizes="32x32" href="{{ site.url }}/assets/img/{{ site.favicon-32x32 }}">{% endif %}
+	{% if site.favicon-32x32 %}<link rel="icon" sizes="32x32" href="assets/img/{{ site.favicon-32x32 }}">{% endif %}
```
Comment (ewan-klein) [inline _includes/_favicon.html]: Can you explain what the advantage is of making this a relative link?

File: _includes/_head.html [hunk 2]
```diff
@@ -22,7 +22,7 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<title>{{ title }}</title>
 
-    <link rel="stylesheet" type="text/css" href="{{ url }}/assets/css/styles_feeling_responsive.css">
+    <link rel="stylesheet" type="text/css" href="assets/css/styles_feeling_responsive.css">
```
Comment (ewan-klein) [inline _includes/_head.html]: Same point: I don't think there is any advantage in replacing the absolute URL with a relative one

File: _includes/_favicon.html (hunk 1 above)
Comment (vradu10) [reply _includes/_favicon.html]: This was taking the logo images from the website so couldn't test without this change. I think relative links are better to make the site portable. I don't see any disadvantage of using relative links.

File: _includes/_head.html (hunk 2 above)
Comment (vradu10) [reply _includes/_head.html]: The same thing here, it was building the static page with CSS from the remote server. I can change it back if you think absolute URLs are better.

File: _includes/_favicon.html (hunk 1 above)
Comment (ewan-klein) [reply _includes/_favicon.html]: If you use the 'local' target in the Makefile, i.e. `% make local`, then `_config_dev.yml` overwrites the setting of `site.url` and you will get images served from the local filesystem.  

I would say that the existing approach, with absolute URLs, isn't broken, so I'd prefer not to change it.

Comment (vradu10) [reply _includes/_favicon.html]: Works well with `make local`. I'm changing it back to relative links.

**Comment by ewan-klein** [comment]
Thanks @vradu10

**Pull request merged**